svn/common.py

```diff
@@ -102,13 +103,13 @@ def info(self, rel_path=None):
         info = {
             'url': root.find('entry/url').text,
 
-            'relative_url': relative_url.text \
-                                if relative_url is not None and \
-                                   len(relative_url.text) \
-                                else None,
+            'relative_url': relative_url.text
+            if relative_url is not None and
+            len(relative_url.text)
+            else None,
 
-# TODO(dustin): These are just for backwards-compatibility. Use the ones added 
-#               below.
+            # TODO(dustin): These are just for backwards-compatibility.
```

Comment:
@dsoprea but this TODO is not a TODO... Is like a warning. :cry: 

Maybe delete this comment and change by a message in README.md... and in a mayor release... drop the backwards compatibility.